src/js/preshuffle.s

```diff
@@ -1202,7 +1213,12 @@ UpdateEnemyHPDisplayInternal:
         jsr StageNametableWriteFromTable
         lda #ENEMY_NAME_VRAM_UPDATE
         jsr StageNametableWriteFromTable
-        jmp @Exit ; early exit
+@EarlyExit:
```

Comment:
This is identical?  Why duplicate the code?  I get that the relative jump isn't able to reach it, but an extra jump only adds ~7 cycles once per many frames, so it doesn't seem worth optimizing away?